src/overlays/actors/ovl_Bg_Jya_Megami/z_bg_jya_megami.h

```diff
@@ -4,11 +4,28 @@
 #include <ultra64.h>
 #include <global.h>
 
+#define BGJYAMEGAMI_NUMPIECES 13
```

Comment:
I'm not sure if there's a naming scheme for defines like this, also it seems weird to me to have it in the header, but I do need it for the array in the actor struct.

Reply:
Using `ARRAY_COUNT(this->pieces)` throughout the file is probably better than using a define.